docs/advanced/migrate-blobs/README.md

```diff
@@ -1,6 +1,6 @@
 # Migrate Cloud Foundry blobs from an NFS blobstore to an Azure Storage blobstore
 
-[`goblob`](https://github.com/pivotal-cf/goblob) is a tool for migrating Cloud Foundry blobs from one blobstore to another. Presently it only supports migrating from an NFS blobstore to an S3-compatible one. [Minio Gateway](https://docs.minio.io/docs/minio-gateway-for-azure.html) adds S3 compatibility to Azure Blob Storage. With these two tools, you can refer to the following steps to migrate blobs from an NFS blobstore to an Azure Storage blobstore. But for a production environment, you need to ask for help from Pivotal support.
+[goblob](https://github.com/pivotal-cf/goblob) is a tool for migrating Cloud Foundry blobs from one blobstore to another. You can refer to the following steps to migrate blobs from an NFS blobstore to an Azure Storage blobstore. But for a production environment, you need to ask for help from Pivotal support.
```

Comment:
The steps are similar to https://github.com/pivotal-cf/goblob#migrate-nfs-blobstore-to-azure-blob-storage. And it seems that goblob doc has more details.
How about adding a link to the goblob doc and remove other contents from this doc? E.g. Only keeping the following contents:

[goblob](https://github.com/pivotal-cf/goblob) is a tool for migrating Cloud Foundry blobs from one blobstore to another. You can [install goblob from source](https://github.com/pivotal-cf/goblob#install-from-source) on your NFS VM, and [migrate blobs from an NFS blobstore to an Azure Storage blobstore](https://github.com/pivotal-cf/goblob#migrate-nfs-blobstore-to-azure-blob-storage). But for a production environment, you need to ask for help from Pivotal support.

Reply:
For people who are not familiar with golang, this document has more detail about how to build goblob from source.